### DISCUSSION TITLE
Disable ctrl+d with specific window title

### DISCUSSION THREAD

**Discussion by miranyx** [kovidgoyal/kitty#6029] [Q&A] [answered]
Hello,

I would like to disable ctrl + d but only for one specific shell which has a custom window title could I somehow disable it for just that shell?

**Comment by kovidgoyal** [comment, marked as answer]
On Wed, Feb 15, 2023 at 10:05:57PM -0800, benni347 wrote:
> Hello,
> 
> I would like to disable ctrl + d but only for one specific shell which has a custom window title could I somehow disable it for just that shell?

ctrl+d is something your shell reacts to not kitty. Simpy unmap it in
your shell config. As for doing it in kitty, no, kitty has no per-window
keymaps.
